backend/app/main.py

```diff
@@ -35,9 +35,11 @@
 
 origins = [
     "http://localhost:3000",
+    "http://127.0.0.1:3000",
```

Comment:
Why do we want to include this one?

Reply:
@remg1997 I don't know which one between localhost and 127.0.0.1 made it worked on local